lib/src/chart/scatter_chart/scatter_chart_data.dart

```diff
@@ -3,9 +3,9 @@ import 'dart:ui';
 
 import 'package:equatable/equatable.dart';
 import 'package:fl_chart/fl_chart.dart';
+import 'package:fl_chart/src/extensions/color_extension.dart';
```

Comment:
This change is unnecessary (It was just removed from line 8 and added in line 6). Please revert all changes like this.

Reply:
@imaNNeoFighT  I think it was auto formatted. I'll check it again.

Reply:
Thank you!

Reply:
@imaNNeoFighT  I'm certain that it's because of the dart format. Do you still want me to revert it?

Reply:
It's better to have features and format changes in different PRs. (Or maybe different commits)

Reply:
ok. I'll change it back.

Reply:
Thank you!

Reply:
I'm unable to change it, as it gets auto formatted immediately.